citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/AsyncUpdateCitationsProcessor.java

```diff
@@ -30,4 +30,12 @@
             ExecutionException, JsonProcessingException {
         return new AsyncResult<ZoteroUpdateItemsStatuses>(zoteroManager.updateCitations(user, groupId, citations));
     }
+    
+    @Async
+    public Future<ZoteroUpdateItemsStatuses> moveCitationsToCollection(IUser user, String groupId, List<ICitation> citations)
```

Comment:
Moving citations to a collection used the batch update method previously. I have separated the batch update method from moving citations to a collection since moving citations to collection does not require creating or updating the metadata note while updating a citations would (if it is used in future, potentially).

Reply:
Is this fine or should I revert back to original code?

Reply:
@jdamerow This change was to be reviewed